legend-depot-store-mongo/src/test/resources/data/versioned-entity-dot-character.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "versionId": "2.2.0",
+    "groupId": "examples.metadata",
+    "artifactId": "test",
+    "versionedEntity": false,
+    "entity": {
+      "path": "examples::metadata::test::TestProfile",
+      "classifierPath": "meta::pure::metamodel::extension::Profile",
+      "content": {
+        "_type": "profile",
+        "name": "dataProfile.profile",
+        "package": "examples::metadata::test"
```

Comment:
Can we add here a key with ".". We have a value with "." but i think there are different rules for keys and values.

Reply:
The error we found was not in key, but the value itself, the object mapper serialization made it a key